test/jdk/javax/swing/JList/TestOpaqueListTable.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import javax.swing.*;
+
+/**
+ *  @test
+ *  @bug 8253266
+ *  @summary setUIProperty should work when opaque property is not set by
+ *  client
+ *  @key headful
+ *  @run main TestOpaqueListTable
+ */
+
+public class TestOpaqueListTable {
+
+    public static void main(String[] args) throws Exception {
+        UIManager.LookAndFeelInfo[] installedLookAndFeels;
```

Comment:
I did not look close to the fix, but you need to check that the default value of the "opaque" property will not be changed. So all default L&F will set it to true, as before the fix.

Reply:
Yeah, the default value for "opaque" property will not be changed.
It will be true for All L&F after the fix as well , I will add  that check to the test.